go/arrow/array/interval.go

```diff
@@ -95,6 +95,9 @@ func (a *MonthInterval) getOneForMarshal(i int) interface{} {
 	return nil
 }
 
+// MarshalJSON will create a json array out of a MonthInterval array,
```

Comment:
not sure if you saw my last comment what do you think about thins`{months=num}`

Reply:
I saw the comment, I wasn't feeling too strongly one way or the other but sure, I'll bump it to that format since it's easy enough to do. After everything passes I can commit this, if you're willing to set your review to approve the changes :)

Reply:
latest commit changes the format to `{"months": #}` and I updated the comments to reflect that in the documentation.